regtech_mail_api/api.py

```diff
@@ -75,17 +75,17 @@ async def get_debug_info(request: Request):
 @app.post("/send")
 @requires("authenticated")
 async def send_email(request: Request):
-    headers = request.headers
-    subject = headers["X-Mail-Subject"]
-    sender_addr = headers["X-Mail-Sender-Address"]
-    sender_name = headers["X-Mail-Sender-Name"]
+    sender_addr = request.user.email
+    sender_name = request.user.name
+    subject = f"SBL Portal User Request for {sender_name if sender_name else request.user.username}"
```

Comment:
I wonder if it's better to leave off the name altogether if it's not available. It probably depends on what's auto-created or not on the case management backend. If a human is always going to have to manually remove that to fill in a real human name once they know it, may be better to pass nothing.

This also gets back to whether we should use another mechanism to provide the sender's contact info. It's not clear to me if SBL Help folks would want a different subject for each of these cases, or if it's better grouping/organizing-wise to have them all match exactly. And using a standard email field that already has that  required-email-address/optional-human-name functionality built-in may be better. I see `ReplyTo:` is an option. If there's not any special magic built-in to that, seems like a good candidate. And we _can_ send custom `X-*` headers as well, so we _could_ even do something similar to the functionality we have in _this_ system.

I think it's all worth a chat with the enterprise dev and our SBL Help team to find out what'd work best for them.

Reply:
Sounds good.  Yeah I mainly put this in just to get, at least currently proposed, the template for frontend testing.  But yes we could have the frontend send user information in headers and if that isn't there, use what's in the token, or vice versa.   Sounds like a meeting is in order perhaps on Monday?  I'll be gone Wed-Fri of next week.

Reply:
Ok this was updated to look for the header "case-type" and will build out a subject like `"[DEV BETA] SBL Portal User Request for <case_type> by <user name>"`. If a user name doesn't exist in the token the subject will lack the `'by <user name>'` part.